java/clients/src/main/java/sleeper/status/report/compaction/task/StandardCompactionTaskStatusReporter.java

```diff
@@ -41,7 +41,7 @@ public class StandardCompactionTaskStatusReporter implements CompactionTaskStatu
     private static final TableField FINISH_TIME = TABLE_FACTORY_BUILDER.addField(StandardProcessRunReporter.FINISH_TIME);
     private static final TableField DURATION = TABLE_FACTORY_BUILDER.addField(StandardProcessRunReporter.DURATION);
     private static final TableField JOB_RUNS = TABLE_FACTORY_BUILDER.addNumericField("JOB_RUNS");
-    private static final TableField JOB_DURATION = TABLE_FACTORY_BUILDER.addNumericField("JOB_DURATION (s)");
+    private static final TableField JOB_DURATION = TABLE_FACTORY_BUILDER.addNumericField("JOB_DURATION");
```

Comment:
Should this just be addField now?

Reply:
We could, although it would left justify the field. Not sure if that would look good if the next fields are numerics and are right justified?
`| 1m       |  1,000,600 |       500,300 |`
vs
`|       1m |  1,000,600 |       500,300 |`

Reply:
Thanks for showing the difference, either way is fine for me.